dataland-frontend/tests/e2e/fixtures/lksg/LksgDataFixtures.ts

```diff
@@ -116,7 +116,7 @@ export function generateLksgData(undefinedProbability = 0.5): LksgData {
       },
       productionSpecific: {
         manufacturingCompany: valueOrUndefined(randomYesNo(), undefinedProbability),
-        capacity: valueOrUndefined(randomNumber(100000), undefinedProbability),
+        capacity: valueOrUndefined(randomNumber(100000).toString(), undefinedProbability),
```

Comment:
Isn't this an actual string already? Should we implement a faker that can produce a number with unit as a string? Because just using a string faker wont create any more realistic fake fixtures, I think.

Reply:
Sorry I did not explain what we mean very well.
Siegfried and I think that the fake values for `capacity` should (also) contain actual words, because we expect the real-world-values to be smth like "2000 cars per hour" or "three tanks per month".

Reply:
changed